cache/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
Here, we don't need to use a `.gitignore`, to keep a folder, you should use a `.gitkeep`

Reply:
Ok done